cms/djangoapps/contentstore/management/commands/generate_courses.py

```diff
@@ -0,0 +1,158 @@
+"""
+Django management command to generate a test course in a specific modulestore
+"""
+import json
+import logging
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+from contentstore.management.commands.utils import user_from_str
+from contentstore.views.course import create_new_course_in_store
+from xmodule.course_module import CourseFields
+from xmodule.fields import Date
+from xmodule.modulestore.exceptions import DuplicateCourseError
+from xmodule.tabs import CourseTabList
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """ Generate a basic course """
+    help = 'Generate a course with settings on studio'
+
+    default_user = "edx@example.com"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'courses',
+        )
+
+    def handle(self, *args, **options):
+        if not settings.DEBUG:
```

Comment:
I'm actually unfamiliar with sandboxes. From a technical standpoint, are they considered dev/local environments? 
I initially added this logic because I told someone that this command was not intended to be run in production. I'm unsure if this was actually necessary. In its current state, it is just a generic command to generate courses... should there be a requirement within the command as to where it can be called?